dash-pipeline/Makefile

```diff
@@ -238,7 +240,7 @@ kill-switch:
 # SAI-THRIFT SERVER TARGETS
 ###############################
 
-saithrift-server:
+saithrift-server: docker-saithrift-bldr
```

Comment:
This technique (as used [here](https://github.com/chrispsommers/DASH/blob/eced43dbd14aaed2f220225cf8eb3329fc629928/dash-pipeline/Makefile#L401)) might eliminate redundant builds:
```
docker-saithrift-client-bldr-image-exists:
	docker images --format "{{.Repository}}:{{.Tag}}" |grep $(DOCKER_SAITHRIFT_CLIENT_BLDR_IMG) || make docker-saithrift-client-bldr
...
docker-saithrift-client: docker-saithrift-client-bldr-image-exists
```

Reply:
incorporated

Reply:
I don't see it, did you push a change?